css-custom-properties/index.css

```diff
@@ -0,0 +1,186 @@
+:root {
+  /********************************** Resets **********************************/
+  margin: 0;
+  border: 0;
+  padding: 0;
+
+  /*************************** Page-wide variables ****************************/
+
+  /* Base spacing unit. */
+  --spacing-unit: 6px;
+
+  /* Margin size. No unit, because it's a multiple of the spacing unit. */
+  --margins: 2;
+
+  /* Colors. */
+  --primary-color: #5E35B1;
+  --primary-color-text: #FFF;
+
+  /* Number of columns to show. */
+  --grid-columns: 3;
+
+  /***************************** Calculated values ****************************/
+  /* We don't use calc here because we don't want to resolve the values yet.
+     You can think of these as simple textual replacements. */
+
+  /* The size of the margin around the card grid. */
+  --margin-size: (var(--margins) * 2);
+  /* How much internal padding each cell should have */
+  --cell-padding: (4 * var(--spacing-unit));
+  /* How big the space between cells should be */
+  --grid-gutter: (var(--margins) * var(--spacing-unit));
+  /* How big the space should be around the grid */
+  --grid-margin: (var(--margin-size) * var(--spacing-unit));
+  /* Calculated cell margin */
+  --cell-margin: (var(--grid-gutter) / 2);
+}
+
+
+body {
+  /* Resets */
```

Comment:
Could you exclude this? We've got a consistent style for `<body>` defined at https://github.com/GoogleChrome/samples/blob/gh-pages/styles/main.css#L1